lms/djangoapps/student_account/test/test_views.py

```diff
@@ -13,12 +13,15 @@
 from django.core.files.uploadedfile import SimpleUploadedFile
 from django.core.urlresolvers import reverse
 from django.contrib import messages
+from django.contrib.auth.models import User
```

Comment:
This is no longer the correct way to retrieve the User model. See https://docs.djangoproject.com/en/1.8/topics/auth/customizing/#referencing-the-user-model.